docs/setup/self-hosted/docker-desktop.mdx

```diff
@@ -25,10 +25,22 @@ Go to the Extensions page in Docker Desktop and search for MindsDB.
 
 Install the MindsDB extension.
 
+<Tip>
+The first time the extension is installed, it will run the latest version of MindsDB. Moving forward, it's advisable to regularly update the MindsDB image used by the extension to ensure access to the latest features and improvements. 
+
+As mentioned previously, the extension uses the `mindsdb/mindsdb:latest` Docker image. To update the image, follow these steps:
+1. Navigate to the 'Images' tab in Docker Desktop.
+2. Search or locate the mindsdb/mindsdb:latest image.
+3. Click on the three dots on the right side of the image and click 'Pull'. If the image is already up to date, you will see a message stating so and you can skip the next step.
+4. Wait for the image to be pulled and restart Docker Desktop.
+
+<img src="/assets/docker/docker_desktop/pull-latest-image.png"/>
+</Tip>
+
 Access MindsDB inside Docker Desktop.
 
 <p align="center">
-  <img src="/assets/mindsdb_docker_desktop.png"/>
+  <img src="/assets/docker/docker_desktop/mindsdb_docker_desktop.png"/>
 </p>
 
 ## Install dependencies
```

Comment:
Please change `##` to `###` for these sections: `Install MindsDB`, `Install dependencies`, and `View logs`.

Reply:
@martyna-mindsdb Done. I also update the Install MindsDB header to Install the MidnsDB Docker Desktop Extension. I hope this is OK?